src/shared/store/Photos/photoSlice.ts

```diff
@@ -5,53 +5,42 @@ import { createAsyncThunk } from '@reduxjs/toolkit';
 import { createSlice } from '@reduxjs/toolkit';
 import API from 'core/services/API';
 
-export const retrievePhotos = createAsyncThunk(
-  'photos/retrieve',
-  async (ids: number[], { rejectWithValue }) => {
-    try {
-      const res = await API.get(`/api/photos${ids.length > 0 ? `?ids=${ids.join()}` : ''}`) as IPhoto[] | IPhoto;
-      return res;
-    } catch (error) {
-      return rejectWithValue(error);
-    }
+export const retrievePhotos = createAsyncThunk('photos/retrieve', async (ids: number[], { rejectWithValue }) => {
+  try {
+    const res = (await API.get(`/api/photos${ids.length > 0 ? `?ids=${ids.join()}` : ''}`)) as IPhoto[] | IPhoto;
```

Comment:
You should also specify in which case it is `IPhoto` and in which it is `IPhoto[]`

Reply:
I changed, but I going to ask about it tomorrow at the lessen